uploadgui/src/main/scala/se/lu/nateko/cp/meta/upload/FormComponents.scala

```diff
@@ -99,6 +104,14 @@ class Radio(elemId: String, cb: String => Unit) {
 	}
 }
 
+class FormTypeRadio(elemId: String, cb: String => Unit) extends Radio(elemId, cb) {
```

Comment:
`cb` constructor argument should have type `FormType => Unit`, not `String => Unit`. `onFileTypeSelected` in Form.scala should be changed accordingly, and not access `typeControl`.